README.rst

```diff
@@ -98,13 +98,21 @@ Quick Example
 
     bot = commands.Bot(command_prefix='$')
 
+
     @bot.command()
     async def ping(ctx):
         await ctx.reply('Pong!')
 
+    @bot.command()
+    async def say(ctx, *, arg):
+         await ctx.message.delete()
+         await ctx.send(f"{ctx.author.mention}'s message: {arg}")
+
     bot.run('token')
 
 
+It is recommended that you use commands.Bot version, as although both client (nextcord.Client) and commands.Bot have the same stuff, commands.Bot is easy to use rather than the client version of nextcord. Make sure to not use both versions though as it can can be confusing and conflicts with your code upon using both in your code.
```

Comment:
This explanation hardly explains why `commands.Bot` is used over `nextcord.Client`.

`nextcord.Client` is fine for bots that don't need message-based commands, Cogs, listeners, or the built-in error handling. You are right that `commands.Bot` makes certain things easier than `nextcord.Client` and that you should not use both. I don't think it is necessary to explain that here though.